rules/scala/private/core.bzl

```diff
@@ -136,18 +136,28 @@ def runner_common(ctx):
 
     args = ctx.actions.args()
     if hasattr(args, "add_all"):  # Bazel 0.13.0+
```

Comment:
(Either Args is documented wrong, or 0.13.0 is still not new enough to match the docs.)